src/io.rs

```diff
@@ -169,6 +170,66 @@ pub fn thread_data_conllx(
     Ok((mmap, start))
 }
 
+/// Meta information about training.
+///
+/// Holds
```

Comment:
Looks like I never finished writing that sentence?

Reply:
Removed the stray *Holds*.